src/test/webhook.spec.ts

```diff
@@ -150,21 +180,108 @@ describe("User testing", function () {
     it("should not delete webhook with invalid ID", async function () {
       const response = await chai
         .request(app)
-        .delete(`/webhooks/${process.env.webhookId}hehe`)
+        .delete(`/webhooks/${webhookId}hehe`)
         .set("content-type", "application/json")
-        .set("Authorization", `${process.env.token}`);
+        .set("Authorization", `${token}`);
 
       expect(response).to.have.status(StatusCodes.BAD_REQUEST);
     });
 
     it("should not delete webhook for unauthenticated user", async function () {
       const response = await chai
         .request(app)
-        .delete(`/webhooks/${process.env.webhookId}`)
+        .delete(`/webhooks/${webhookId}`)
         .set("content-type", "application/json")
-        .set("Authorization", `${process.env.token}hehe`);
+        .set("Authorization", `${token}hehe`);
 
       expect(response).to.have.status(StatusCodes.UNAUTHORIZED);
     });
   });
+
+  describe("Request interception", function () {
```

Comment:
This should describe what are you testing in this suite, and it's not testing `Request interception`.
Request interception is what you're doing to test that webhook notifications are sent through the webhook service.